aloha-core/src/main/scala/com/eharmony/aloha/semantics/compiled/plugin/schemabased/tokenization/SchemaBasedTokenizer.scala

```diff
@@ -0,0 +1,68 @@
+package com.eharmony.aloha.semantics.compiled.plugin.schemabased.tokenization
+
+
+import scala.util.parsing.combinator.JavaTokenParsers
+import scalaz.ValidationNel
+import scalaz.syntax.validation.ToValidationV // scalaz.syntax.validation.ToValidationOps for latest scalaz
```

Comment:
Should we upgrade the `scalaz` version at this time or consider moving to `cats`?